README.md

```diff
@@ -117,22 +97,15 @@ copies of the same item).
 
 These raw files are "scrubbed" and verified, resulting in scrubbed files.
 
-To load a scrubbed file:
+To load a scrubbed file in development:
   * Get the file(s) you want from `/htapps/mwarin.babel/phdb_scripts/data/loadfiles/`
   * Add UUIDs for tracking whether the individual line has been processed: `bin ruby/add_uuid.rb infile > outfile`
   * `docker-compose run --rm dev bundle exec bin/add_print_holdings.rb outfile`
 
-
-
 ## K8s Cronjob
 `kubectl create -f cron_job.yaml`
```

Comment:
OT: I don't think this ever made it from the old concordance repo to holdings-backend. The cronjob in ht_tanka remains commented out. https://github.com/hathitrust/ht_tanka/pull/9/files#diff-d37db7a43061041cef4bba8b70897d4bdaa3b8675e491a31c89a56a395295de6

Reply:
Thanks -- I was wondering where that ended up. I was thinking we were waiting to re-enabled it until we fully automate the whole process -- do we want it enabled now?